packages/react/src/SubdomainNavBar/SubdomainNavBar.module.css

```diff
@@ -535,7 +545,7 @@
   }
 }
 
-@media screen and (min-width: 48rem) {
+@media screen and (min-width: 767px) {
```

Comment:
What was the reason for this change?

FYI, we prefer to use `rem` values for breakpoints... here's the available ones:

![Screenshot 2023-10-12 at 09 21 33](https://github.com/primer/brand/assets/13340707/26d8f6fe-5ca6-4204-8a2c-dad168c075bd)

Reply:
I can drop down to `47.75rem`, but I had to change this to fix a bug where **both** the CTA buttons and the sandwich menu were invisible at exactly `768px`.

Reply:
perhaps `calc(var(--brand-breakpoint-medium) - 1px)`?